src/main/resources/gnss_log_2021_02_05_13_20_58.txt

```diff
@@ -0,0 +1,345 @@
+Fix,gps,28.064160,-82.419935,-18.411621,0.000000,4.812984,1612549548000
+Fix,gps,28.064160,-82.419936,-18.130096,0.170000,4.878742,1612549549000
+Fix,gps,28.064162,-82.419940,-18.376434,0.790000,4.598483,1612549550000
+Fix,gps,28.064161,-82.419953,-18.367828,1.770000,4.234781,1612549551000
+Fix,gps,28.064160,-82.419973,-18.469574,1.920000,3.929803,1612549552000
+Fix,gps,28.064161,-82.419999,-18.494171,2.550000,3.796088,1612549553000
+Fix,gps,28.064161,-82.420029,-18.306671,3.170000,3.797549,1612549554000
+Fix,gps,28.064156,-82.420064,-18.157227,3.810000,3.792729,1612549555000
+Fix,gps,28.064149,-82.420101,-18.695099,4.130000,3.827508,1612549556000
+Fix,gps,28.064131,-82.420142,-18.931396,4.520000,3.801275,1612549557000
+Fix,gps,28.064105,-82.420179,-18.883545,4.620000,3.808761,1612549558000
+Fix,gps,28.064069,-82.420213,-19.109497,5.270000,3.799585,1612549559000
+Fix,gps,28.064024,-82.420236,-19.303528,5.690000,3.793882,1612549560000
+Fix,gps,28.063968,-82.420251,-18.843567,6.150000,3.801342,1612549561000
+Fix,gps,28.063909,-82.420257,-19.238037,6.620000,3.813566,1612549562000
+Fix,gps,28.063852,-82.420258,-19.413757,6.990000,3.794435,1612549563000
+Fix,gps,28.063783,-82.420255,-19.337494,7.410000,3.796766,1612549564000
+Fix,gps,28.063714,-82.420249,-19.433167,7.720000,3.821010,1612549565000
+Fix,gps,28.063645,-82.420236,-19.155853,7.870000,3.817261,1612549566000
+Fix,gps,28.063576,-82.420222,-18.591614,8.010000,3.805329,1612549567000
+Fix,gps,28.063504,-82.420197,-17.032898,8.180000,3.806547,1612549568000
+Fix,gps,28.063431,-82.420162,-17.205414,8.280000,3.816261,1612549569000
+Fix,gps,28.063362,-82.420121,-17.330597,8.350000,3.825449,1612549570000
+Fix,gps,28.063291,-82.420071,-16.878967,8.770000,3.799215,1612549571000
+Fix,gps,28.063225,-82.420022,-16.859039,8.830000,3.828022,1612549572000
+Fix,gps,28.063158,-82.419971,-16.441132,8.850000,3.861558,1612549573000
+Fix,gps,28.063093,-82.419918,-16.306000,8.830000,3.819880,1612549574000
+Fix,gps,28.063029,-82.419869,-15.742706,8.780000,3.884212,1612549575000
+Fix,gps,28.062962,-82.419821,-14.616394,8.720000,3.866363,1612549576000
+Fix,gps,28.062891,-82.419770,-14.727722,8.830000,3.931311,1612549577000
+Fix,gps,28.062822,-82.419728,-15.412170,8.740000,3.956826,1612549578000
+Fix,gps,28.062753,-82.419689,-15.302856,8.750000,4.228830,1612549579000
+Fix,gps,28.062679,-82.419653,-14.909729,8.690000,4.636364,1612549580000
+Fix,gps,28.062606,-82.419621,-15.067780,8.620000,5.187472,1612549581000
+Fix,gps,28.062534,-82.419591,-14.950562,8.580000,4.966069,1612549582000
+Fix,gps,28.062463,-82.419562,-15.047394,8.650000,4.327048,1612549583000
+Fix,gps,28.062388,-82.419536,-15.274597,8.670000,4.037940,1612549584000
+Fix,gps,28.062312,-82.419516,-15.473328,8.610000,3.854686,1612549585000
+Fix,gps,28.062237,-82.419498,-15.243195,8.590000,3.810133,1612549586000
+Fix,gps,28.062166,-82.419487,-15.628876,8.410000,3.805122,1612549587000
+Fix,gps,28.062087,-82.419479,-15.686646,8.30
```

Comment:
Looks like this record got cut off (likely being written during transfer). it's actually a good test to make sure we can handle it without crashing.